utils/extensions.py

```diff
@@ -0,0 +1,74 @@
+import json
+import os
+
+from utils.launch import run
+
+extensions_dir = "extensions"
+git = os.environ.get("GIT", "git")
+
+
+def opener(path, flags):
+    dir_fd = os.open("extensions/config", os.O_RDONLY)
+    return os.open(path, flags, dir_fd=dir_fd)
+
+
+def download_extensions():
```

Comment:
This is the bread and butter on how I'd like for us to import any extensions we add to the `stable-diffusion-extensions` GitHub repo.